js/classes/inventory.js

```diff
@@ -46,11 +51,16 @@ class Inventory {
     }
 
     /**
-     * @returns {number} The amount of ammo the player currently has.
+     * @returns {Ammo} The ammo that was just used. None if the ammo 
```

Comment:
I may be misreading it but it seems like the @return tag comment is incomplete. Additionally, it looks like you are returning a string if the ammo type is empty and an integer Projectile type otherwise, never an Ammo type.